webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+import statistics from './statistics';
+
+export default combineReducers({
```

Comment:
this is only the root of the state tree.. it may grow, but not much, as the leafs may also export "combineReducers"